lib/fhir/resources/encounter.js

```diff
@@ -94,7 +38,20 @@ module.exports = (mongo) => {
         }
 
         let build = (practitionersToFilter) => {
-          let query = buildSearchFilters(queryObject, practitionersToFilter)
+          const filteredQueryObject = queryUtils.filterOutProperties(queryObject, Object.keys(customSupportedParams))
+          const query = queryBuilder.buildQuery(ctx.resourceType, filteredQueryObject)
+
+          if (practitionersToFilter) {
```

Comment:
I wonder if the `practitioner.organization` query using `practitionersToFilter` could rather be done using a chained parameter query. This would make more sense I think and use for generic code.